setup.py

```diff
@@ -18,10 +18,10 @@
       ],
       py_modules=['tap_mysql'],
       install_requires=[
-          'attrs==16.3.0',
+          'attrs==19.3.0',
```

Comment:
attrs is already installed by `pipelinewise-singer-python`, so we don't need to re-install it again. wdyt?

Reply:
yes, thanks. 🙇 
Fixed by 143e398795a69047f45b7f44bd66f57b0573bdb4